.github/workflows/terraform.yml

```diff
@@ -28,6 +28,11 @@ jobs:
     - name: Checkout the repository to the GitHub Actions runner
       uses: actions/checkout@v2
 
+    - name: Use Terraform 0.12.29
```

Comment:
This is not required in this PR as statuscake is not currently delivered via github actions. We should create a separate card and PR for that.